content/fr/docs/5.configuration-glossary/6.configuration-components.md

```diff
@@ -1,21 +1,266 @@
 ---
-title: La Propriété Composants
+title: La propriété components
 navigation.title: components
 description: 'Nuxt 2.13+ peut scanner et importer automatiquement vos composants en utilisant le module @nuxt/components'
 menu: components
 category: configuration-glossary
 ---
-# La Propriété Composants
+# La propriété Composants
 
 Nuxt 2.13+ peut scanner et importer automatiquement vos composants en utilisant le module @nuxt/components.
 
 ---
 
-- Type: `Boolean` or `Object`
+::alert
+Il est possible d'utiliser cette fonctionnalité avec Nuxt 2.10 - 2.12. Installez manuellement en ajoutant `@nuxt/components` dans `buildModules` à l'intérieur de `nuxt.config`.
+::
+
+- Type: `Boolean` ou `Object`
 - Default: `false`
 
 Lorsqu'il est défini à `true` ou qu'il utilise un objet, il inclura les dépendances du [composant](https://github.com/nuxt/components) et importera automatiquement vos composants (définis dans `~/components`) lorsque vous les utilisez dans vos modèles.

```

Comment:
should we translate "component" to "composant" everywhere ?

Reply:
It seems more logical to put the same word everywhere

Reply:
In a sentence, yes we need to translate it but not for references.